Content.Shared/Cognislime/CognislimeComponent.cs

```diff
@@ -38,6 +38,8 @@ public sealed partial class CognislimeComponent : Component
         Components = new[]
         {
             "PAI",
+            "MindContainer",
```

Comment:
pai has mindcontainer so it isnt needed
and im pretty sure both mmi and posibrain have MindContainer by default

Reply:
I tested it and posibrain seems to not have mindcontainer by default, but i will get rid of the pai blacklist

Reply:
Looks like pAIs don't have mindcontainer by default either so back in they go